### ISSUE TITLE
tensorflow.python.famework.errors_impl.FailedPreconditionError 

### ISSUE THREAD

**Issue by jaehwlee** [tae-jun/resemul#5]
Hello! Thank you so much for revealing the code.

I tried to add a user-defined layer to the front layer to increase the accuracy of this code. 

However, an error occurred because the user-defined code was written in Keras 2.1.6

![image](https://user-images.githubusercontent.com/33409264/91436281-c07d3b00-e8a2-11ea-9239-61387dfcf94d.png)

In fact, the error has disappeared, but I don't know if this is the right solution. In addition, I want to let people know who have the same worries as me.

* My development environment is as follows
Tensorflow : 1.15
Keras : 2.0.5

**Comment by jaehwlee** [comment]
The error occurred here.
import keras.backend as K
y = K.concatenate([y_left, K.variable(K.ones(1)), y_right])

I modify this as follows.
y = K.concatenate([y_left, K.ones((1,)), y_right])

Let me know if anyone has a better solution :)

**Comment by jaehwlee** [comment]
I solved it with appropriate method.
a = K.variable(np.array([1])
y = K.concatenate([y_left, a, y_right])
